messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/logging/MessagingLogger.java

```diff
@@ -819,4 +819,6 @@
     @Message(id = 87, value = "Unable to load connector service factory class: %s")
     OperationFailedException unableToLoadConnectorServiceFactoryClass(String factroyClass);
 
+    @Message(id = 88, value = "Invalid journal-file-size %s , It should be multiple of 512")
```

Comment:
Thanks. I have made the message more flexible. About message itself, if it's smaller than 1024, it will display message: 

> "WFLYCTL0117: 1021 is an invalid value for parameter journal-file-size. A minimum value of 1024 is required" 

due to validation in super class.